src/net_processing.cpp

```diff
@@ -513,7 +513,7 @@ struct Peer {
     bool m_should_discourage GUARDED_BY(m_misbehavior_mutex){false};
 
     /** Set of txids to reconsider once their parent transactions have been accepted **/
-    std::set<uint256> m_orphan_work_set;
+    std::set<uint256> m_orphan_work_set GUARDED_BY(g_cs_orphans);
```

Comment:
673247b58cd1252ab7e99f7d63ead05cc100cef2, nit:
```suggestion
    std::set<uint256> m_orphan_work_set GUARDED_BY(::g_cs_orphans);
```

Reply:
Thank you for the suggestion! Could you point me towards anything which indicates why this might be preferred? Based on my understanding, it seems like a style preference. 1) Globals are already prefixed with `g_` making it very unlikely to have a conflicting local variable 2) I don't see anywhere else in the codebase using this operator for globals.

Reply:
`git grep ::cs` or https://github.com/bitcoin/bitcoin/blob/f2e6d14430137a271d153348d207df6ab8086bc6/src/init.cpp#L216

> Based on my understanding, it seems like a style preference.

Correct. Feel free to ignore this nit.